scripts/generate-component-docs.rb

```diff
@@ -798,6 +798,17 @@ def resolve_schema(root_schema, schema)
     end
   end
 
+  # required for global option configuration
+  is_common_field = get_schema_metadata(schema, 'docs::common')
```

Comment:
I think it's temporary solution for Ruby docs generator, and I do believe that it would be much bettern when we port Ruby code to Rust.